src/main/kotlin/eu/europa/ec/eudi/openid4vci/NotifyIssuer.kt

```diff
@@ -63,6 +63,6 @@ fun interface NotifyIssuer {
          * @return a [NotifyIssuer]
          */
         internal operator fun invoke(notificationEndPointClient: NotificationEndPointClient): NotifyIssuer =
-            NotifyIssuer { event -> notificationEndPointClient.notifyIssuer(accessToken, event) }
+            NotifyIssuer { event -> notificationEndPointClient.notifyIssuer(accessToken, resourceServerDpopNonce, event) }
```

Comment:
The `notify` methods has to be changed, I think.
Instead of returning a `Result<Unit>`,  it should return the possibly updated `AuthorizedRequest`. Perhaps `AuthorizedRequestAnd<Unit>`